lib/smalloc.js

```diff
@@ -3,4 +3,4 @@
 const util = require('internal/util');
 
 module.exports = require('internal/smalloc');
-util.printDeprecationMessage('smalloc is deprecated.');
+util.deprecate('smalloc is deprecated. Use typed arrays.');
```

Comment:
@targos Oh, you are correct. `printDeprecationMessage` should stay. But, in case of `require`, we can print only once. So, `--trace-deprecation` cannot do much here, right?